OloEngine/tests/Rendering/OcclusionIntegrationTest.cpp

```
@@ -0,0 +1,292 @@
+        glm::translate(glm::mat4(1.0f), glm::vec3(0, 0, -30)), // visible
+        glm::translate(glm::mat4(1.0f), glm::vec3(0, 0, 100)), // behind camera
+        glm::translate(glm::mat4(1.0f), glm::vec3(2, 0, -5)),  // visible
+    };
+
+    // Simulate the per-instance culling logic from DrawMeshInstanced:
+    // For each instance, test its position against the frustum
+    std::vector<glm::mat4> activeTransforms;
+    for (const auto& t : transforms)
+    {
+        glm::vec3 pos = glm::vec3(t[3]); // Extract translation
+        if (frustum.IsPointVisible(pos))
```

Cppcheck (2.19.0)

[style] The function 'ExpectCommandOrder' is never used.

(unusedFunction)